### ISSUE TITLE
Video stops playing under mpv

### ISSUE THREAD

**Issue by colibris79** [smplayer-dev/smplayer#572]
**Describe the bug**
While playing a video it will randomly stop playing. It will not go into "pause", it will just stop. I have to press "Play" twice (or the space bar) in order to continue playing. The first time it will go into pause, the second time it will keep on playing.

**To Reproduce**
I don't know exactly what triggers it. It has been happening while playing mp4/mkv video files with mpv engine and it can happen at any time. With mplayer engine it works fine. Mpv media player works fine

**Expected behavior**
The video should not stop

**Your Enviroment**
 - SMPlayer version:  22.7.0-1-jammy1
 - OS: Mint 21 (Ubuntu 22.04). Using AMD integrated graphic card, amdgpu 22.20.50203
MPV 0.34.1-1

**Logs**
The log didn't show anything special while the error happened, just:
[15:20:36:991] BaseGui::checkStayOnTop

**Comment by colibris79** [comment]
I've tried with smplayer version from repos that is still 21.10 (that it was working fine in another computer with an Intel integrated gpu). I've also updated amdgpu to last 5.3 version. Still having the same problem. Smplayer + mpv (stops randomly)
Mpv player and Smplayer + mplayer work fine
So could it be a gpu related issue?

EDIT: It seems that in the computer giving problems if I disable the compositing (and I leave just Marco for example), the problem doesn't happen. I'll keep an eye on it
